Dockerfile

```diff
@@ -23,7 +23,8 @@ RUN apk add --update ca-certificates
 WORKDIR /root
 
 # Copy over binaries from the build-env
-COPY --from=build-env /go/src/github.com/cosmos/ethermint/build/emintd /usr/bin/emintd
+COPY --from=build-env /go/src/github.com/Chainsafe/ethermint/build/emintd /usr/bin/emintd
+COPY --from=build-env /go/src/github.com/Chainsafe/ethermint/build/emintcli /usr/bin/emintcli
```

Comment:
Just a note, I believe this wasn't included in the past because you would only need to use the CLI to generate a key for the address and you could do all of that locally and only pass in the address for which you would want to allocate for genesis.

I am assuming you need this to run the rest-server that now exists in a container?

Reply:
Yes. `emintcli` is needed for running the rest-server in the container.